src/smpp34pdu_submit_sm.erl

```diff
@@ -55,7 +55,8 @@ pack(#submit_sm{service_type=SrvType,
 		its_reply_type=ItsReplyType,
 		its_session_info=ItsSessionInfo,
 		ussd_service_op=UssdServiceOp,
-                ussd_session_id=UssdSessionId}) ->
+                ussd_session_id=UssdSessionId,
+        vendor_specific = VendorSpecific }) ->
```

Comment:
fix indentation

Reply:
Found out the trick. Github displays tab as 8 spaces for some reason. I set my VSCode to ident code and replace tab with 4 spaces.  
Fixed.